Codeinwp/neve
# Remove gtb templates

### Summary
- Removed the Gutenberg templates for otter.
- Removed the function that was adding them in otter.

### Will affect visual aspect of the product
NO

### Test instructions
- Install Neve and Otter
- Create a new page and try to import a template ( https://d.pr/XdGNpT )
- Before this, the first templates that appeared here were the ones from neve original starter sites, now it should look like this https://vertis.d.pr/7XSyqQ


Closes #2905.
